renderer/actions/FieldActions.ts

```diff
@@ -1,7 +1,7 @@
 import * as consts from '../consts';
 import { FieldActions } from './types';
 
-export const updateTimer: FieldActions['updateTimer'] = (msg: any) => ({
+export const updateTimer: FieldActions['updateTimer'] = (msg: { total_stage_time: number; stage_time_so_far: number; stage_name: string; }) => ({
```

Comment:
@HaileyJang Yes, that is correct, since the action's return type is basically what the reducer will take in for a given action, those types can be shared. I'll do an example real quick.